code/modules/jobs/job_types/chaplain/chaplain_vorpal_scythe.dm

```diff
@@ -19,13 +19,12 @@ If the scythe isn't empowered when you sheath it, you take a heap of damage and
 	var/obj/item/vorpalscythe/scythe = active_item
 	if(!scythe)
 		return FALSE
-	if(scythe.empowerment < SCYTHE_SATED)
+	var/obj/item/bodypart/part = hand
+	if(scythe.empowerment < SCYTHE_SATED && part)
 		to_chat(owner, span_userdanger("[scythe] tears into you for your unworthy display of arrogance!"))
 		playsound(owner, 'sound/magic/demon_attack1.ogg', 50, TRUE)
-		var/obj/item/bodypart/part = owner.get_holding_bodypart_of_item(scythe)
-		if(part)
-			part.receive_damage(brute = 25, wound_bonus = 10, sharpness = SHARP_EDGED)
-	return ..()
+		part.receive_damage(brute = 25, wound_bonus = 10, sharpness = SHARP_EDGED)
+	..()
```

Comment:
Why get rid of `return ..()`?

Reply:
Oh, entirely personal preference there, I just find `..() `to look cleaner than` return ..()` but I don't feel particularly strong about it either way.

Reply:
Generally speaking we prefer `return ..()` unless you explicitly want to discard the return value of parent, just to make it easier on future developers (on the off chance they add a return value or rely on return value of this proc)